docs/pages/setup/admin/github-sso.mdx

```diff
@@ -33,10 +33,17 @@ Instructions for creating a GitHub OAuth app are available here:
 
 [Creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
 
-## Step 2/3. Create a GitHub authentication connector
+## Step 2/5. Define the GitHub authentication connector
 
-Define a GitHub authentication connector by creating a file called `github.yaml`
-with the following content:
+Define a GitHub authentication connector by creating a file called `github.yaml`. 
```

Comment:
```suggestion
Define a GitHub authentication connector by running the following command:
```
I think we can remove the instruction to manually create the `github.yaml` file and condense the first two paragraphs of this section.